FordLabs/PeopleMover
# Co-authored-by: Ashley Clifton <aclift19@ford.com>

Co-authored-by: Andrew Crouch <jcrouc15@ford.com>
Co-authored-by: John Cherney <jcherney@ford.com>
Co-authored-by: Chris Atkinson <christopher.w.atkinson.1989@gmail.com>

## Issue
Resolves 215 Person Tags, Update Unassigned badge to mirror counter element unassigned people count

## What was done
- [x] Rename reusable component Counter to CounterElement
- [x] Create Counter.ts file to contain code to calculate filtered counts
- [x] Update CounterElement to user the new Counter file
- [x] Update UnassignedDrawer to use the new Counter file

## How to test
1. Open PeopleMover
2. Add Products and People, making some people assigned, some unassigned
3. Give Tags to People and Products (Product Tags, People Tags, Location, Role)
4. Filter by the tags. Note that the people in unassigned appear or disappesr as they are filtered, and that their count updates in both the CounterElement and in the Unassigned Drawer badge, and that the counts are identical.
